cylc/flow/xtrigger_mgr.py

```diff
@@ -133,6 +133,13 @@ def __init__(
             TMPL_SUITE_SHARE_DIR: suite_share_dir,
             TMPL_DEBUG_MODE: cylc.flow.flags.debug
         }
+
+        if (proc_pool is None or broadcast_mgr is None or
+                data_store_mgr is None):
+            raise TypeError(
+                "kwargs proc_pool, broadcast_mg and data_store_mgr may not "
+                "be None.")
+
```

Comment:
I think the tests are passing in `None` here.

Reply:
Changed to https://github.com/cylc/cylc-flow/blob/c7af49d4d0de15bb4b499a2fc5fb3b51bea3f7f9/cylc/flow/xtrigger_mgr.py#L137-L142